packages/console/src/components/CreateConnectorForm/index.tsx

```diff
@@ -155,7 +155,7 @@ function CreateConnectorForm({ onClose, isOpen: isFormOpen, type }: Props) {
           <>
             <div className={styles.standardLabel}>
               <DynamicT forKey="connectors.standard_connectors" />
-              {isStandardConnectorDisabled && <ProTag />}
+              <ProTag isVisibleInProdEnv={isStandardConnectorDisabled} />
```

Comment:
Consider changing the name to "ProdTenant"?

Reply:
Yeah, tenant is better than env

Reply:
Updated.